src/Services/FHIR/FhirPatientService.php

```diff
@@ -308,6 +308,13 @@ private function parseOpenEMRPatientTelecom(FHIRPatient $patientResource, $dataR
         if (!empty($dataRecord['email'])) {
             $patientResource->addTelecom($this->createContactPoint('email', $dataRecord['email'], 'home'));
         }
+        if (!empty($dataRecord['email_direct'])) {
+            $patientResource->addTelecom($this->createContactPoint('email', $dataRecord['email_direct'], 'mobile'));
```

Comment:
This seems odd to me.  The email_direct is for email that goes over the secure Direct Network Exchange (a good reading is here: https://www.emrdirect.com/about-directed-exchange-and-secure-direct-messaging.html).  It seems like these communications would be for routine business that needs to be used for transmitting secure health information and not for urgent communications.  I would think we would use 'work' here instead of 'mobile'.

Reply:
Thanks for the link. I had missed that connotation. It should be explained as such in the UI. If it is, I missed that. Without going deeper, seems like there's a registration step implied/involved beyond just an email address. I'll look for that.

With all of that, I think "work" email is the opposite.  For most, that's definitely not a more private/protected channel like "direct" is. I certainly assume my "work" email is scanned, and it should be anywhere, and I work for a terrific guy (hi @mi3guyc). 

"mobile" on the other hand is (assumed) private, and that choice is rarely used with "email" in my experience with other FHIR implementations.  Since the list of choices is short, I still think mobile is logical.

I alluded to collisions on email. The db doesn't support work/home email so the code is/was written to push the last one received into the email field.  That's still an issue not addressed here, but I think it should be someday like other EMR's.

Reply:
I think this is resolved.